cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
@dnephin Exporting a substandard interface is worse that not making it available at all. We had this experience when refactoring the archive package for containerd. Even with exported interfaces, DRY isn't necessarily achieved, unless people go through the process of discovery with godoc or other tools. Duplicating code isn't necessarily a bad thing, especially when it means that the two pieces of code aren't coupled. When the code is ready to be used in multiple places, that is the time to design an exported interface.

Too many Go projects suffer from "over-packagification", including docker. Let's not contribute to it and let's not force it on our contributors.